apps/mobile/src/components/headers/modal-header.tsx

```diff
@@ -9,6 +9,8 @@ import {
   TouchableOpacity,
 } from '@leather.io/ui/native';
 
+import { MODAL_HEADER_HEIGHT } from './constants';
```

Comment:
Should I start moving these to global constants?

Reply:
I moved this `ModalHeader` into `headers` bc I actually created one not realizing a component was in another location, `modal-headers`.

Reply:
I can refactor this. `Modal` is to become `Sheet` and I can locate the header alongside that in `ui/packages/`. 

For the extension, we use some tokens for these types of heights. So we could move them to the tokens? 

Having a localised constants file is OK unless we need to use them in loads of places